src/components/ErrorNotification/ErrorNotification.tsx

```diff
@@ -0,0 +1,32 @@
+import classNames from 'classnames';
+import React from 'react';
+
+import { ErrorMessages } from '../../types/ErrorMessages';
+
+type Props = {
+  message: string;
+  onRemoveError: () => void;
+};
+
+export const ErrorNotification: React.FC<Props> = ({
+  message,
+  onRemoveError,
+}) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        message === ErrorMessages.noError ? ' hidden' : '',
```

Comment:
Check how to use classnames library properly, https://www.npmjs.com/package/classnames